Mage.Sets/src/mage/cards/f/FelisaFangOfSilverquill.java

```diff
@@ -30,14 +31,6 @@
  */
 public final class FelisaFangOfSilverquill extends CardImpl {
 
-    private static final FilterPermanent filter
-            = new FilterControlledCreaturePermanent("a nontoken creature you control");
-
-    static {
-        filter.add(TokenPredicate.FALSE);
-        filter.add(CounterAnyPredicate.instance);
-    }
-
```

Comment:
This filter is checking that the permanent had counters on it even though the filter text doesn't say so (it's harmlessly "cheating" the intervening-if clause rules by folding the clause into the filter, since the truth of the condition can't possibly change between triggering and resolution).